bin/build.sh

```diff
@@ -1,11 +1,17 @@
 #!/bin/sh
-echo "Installing PHP packages..."
-composer install
+COMPOSER_COMMAND='install'
+
+if [ "$1" = "-u" ] || [ "$1" = "-update" ]; then
```

Comment:
With CLI programs, the long options names (more than one letter) always start by two dashes. That way, the program can differentiate the `--update` option from the concatenation of the `-u -p -d -a -t -e -s` options.

Reply:
Done!